### DISCUSSION TITLE
Two-Year%20MSCS%40Yale/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#153] [Announcements]
# Two-Year%20MSCS%40Yale/

https://csmsapp.org/Two-Year%20MSCS%40Yale/

**Comment by dadyita** [comment]
托福严卡26分，因为需要学期内做TA

**Reply by dadyita** [reply]
*托福口语严卡26分

**Reply by KingArthur0205** [reply]
貌似25fall砍funding不招人了，网上pyq都没看到dp？

**Comment by KingArthur0205** [comment]
貌似25fall砍funding不招人了，网上pyq都没看到dp？
